errai-codegen/src/test/java/org/jboss/errai/codegen/meta/impl/apt/APTMethodTest.java

```diff
@@ -17,41 +17,113 @@
 package org.jboss.errai.codegen.meta.impl.apt;
 
 import org.jboss.errai.codegen.apt.test.ErraiAptTest;
+import org.jboss.errai.codegen.meta.MetaClass;
+import org.jboss.errai.codegen.meta.MetaClassFactory;
 import org.jboss.errai.codegen.meta.MetaMethod;
 import org.junit.Assert;
 import org.junit.Test;
 
 import javax.lang.model.type.TypeMirror;
+import java.util.List;
 
 /**
  * @author Tiago Bento <tfernand@redhat.com>
  */
 public class APTMethodTest extends ErraiAptTest {
```

Comment:
Yeah, absolutely. I thought about it when I was writing these tests, but since the next PR will be focused on tests, I think we should address it'll be the place to address that.